HW4/Olenka_M/HomeWork4.2.py

```diff
@@ -0,0 +1,7 @@
+integer_number_list = [1,2,6,7,9,45,-34,0]
+float_number_list = []
+
+for number in integer_number_list:
```

Comment:
for number in integer_number_list:
       float_number_list.append(float(number))
print(float_number_list)